src/cryptoadvance/specter/util/tor.py

```diff
@@ -40,6 +40,24 @@ def start_hidden_service(app):
             f.write("%s.onion" % app.tor_service_id)
     app.tor_service_id = app.tor_service_id
     app.tor_enabled = True
+    if app.specter.config.get("auth", "none") == "none":
```

Comment:
Throw a SpecterError("Sorry, Tor is not supported with no Authentication-method! Please activate one before activating Tor!")
Ideally there is somewhere up the call-stack the endpoint which is flashing it to the user (or an API-call which returns a 500 with that message).

Reply:
But I think from the command line we can allow this, and just block it from the UI, so that advanced users can decide for themselves.